velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -503,5 +503,28 @@ TEST_F(DateTimeFunctionsTest, nextDay) {
   EXPECT_EQ(nextDay("2015-07-23", ""), std::nullopt);
 }
 
+TEST_F(DateTimeFunctionsTest, hour) {
+  const auto hour = [&](const StringView& timestampStr) {
+    const auto timeStamp =
+        std::make_optional(util::fromTimestampString(timestampStr));
+    return evaluateOnce<int32_t>("hour(c0)", timeStamp);
+  };
+
+  EXPECT_EQ(0, hour("2024-01-08 00:23:00.001"));
+  EXPECT_EQ(0, hour("2024-01-08 00:59:59.999"));
+  EXPECT_EQ(1, hour("2024-01-08 01:23:00.001"));
+  EXPECT_EQ(13, hour("2024-01-20 13:23:00.001"));
+  EXPECT_EQ(13, hour("1969-01-01 13:23:00.001"));
+
+  // set time zone to Pacific/Apia (13 hours ahead of UTC)
```

Comment:
Comment starts with a capital letter and ending with a period (.).

Reply:
Thanks! Updated.